.github/copilot-instructions.md

```diff
@@ -0,0 +1,9 @@
+For reviewing PRs:
+* All functions in header files should have doxygen-style API docs
+* Use /// for single-line comments rather than /** */
+* Use meaningful variable names, e.g. `measurement` not `msm`, avoid abbreviations.
```

Comment:
This could be even "enforced" by using clang-tidy if so wanted at some point...

Reply:
I would *love* that :-)